### ISSUE TITLE
[question] crc32 calc way

### ISSUE THREAD

**Issue by ilyar** [tonlabs/ever-struct#2]
```tlb
tvc_none#a8775f2f = TvmSmc;
tvc_frst#ece10b0d code:^Cell meta:(Maybe ^Metadata) = TvmSmc;

tvc#0167f70c tvc:TvmSmc = TVC;

// crc32 <(printf "tvc_none") == a8775f2f
// crc32 <(printf "tvc_frst") == ece10b0d
// crc32 <(printf "tvc")      == 0167f70c
```

vs

```tlb
// crc32 <(printf "tvc_none = TvmSmc") == fa90fdb2
tvc_none#fa90fdb2 = TvmSmc;
// crc32 <(printf "tvc_frst code:^Cell meta:(Maybe ^Metadata) = TvmSmc") == b96aa11b
tvc_frst#b96aa11b code:^Cell meta:(Maybe ^Metadata) = TvmSmc;
// crc32 <(printf "tvc tvc:TvmSmc = TVC") == 8b5f2433
tvc#8b5f2433 tvc:TvmSmc = TVC;
```

**Comment by cryshado** [comment]
@ilyar if we are following the way that `tlbc` calculates constructor tags, then at least it removes `(` and `)` from the schemas. So I propose to use `tlbc` to calculate constructor tags (we can extract it from warnings)

**Comment by cryshado** [comment]
Resolved via #4
